automatic/lightalloy/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,28 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName = 'lightalloy'
+$url32 = ''
+$checksum32  = ''
```

Comment:
not if you change the regex to like `(?i)(^\s*url\s*=\s*)('.*')`
See the [fiddler4](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/automatic/fiddler4/update.ps1#L8-L9) package as an example

Or to say it another way, keep the regex for the url and checksum that you had previously. I only meant that you should change the packageName regex in update.ps1, not the url and checksum regex.